lib/cell.rb

```diff
@@ -0,0 +1,46 @@
+require './lib/ship'
+
+class Cell
+  attr_reader :coordinate, :ship, :fire_upon
+
+  def initialize(coordinate)
+    @ship = nil
+    @coordinate = coordinate
+    @fired_upon = false
+    @render = "."
+  end
+
+  def empty?
+    if self.ship == nil
+      true
+    else
+      false
+    end
+  end
+
+  def place_ship(ship_type)
+    @ship = ship_type
+    self.ship
```

Comment:
This line I think we can remove since @ship = ship_type will return the ship already.